rust-webvr/src/api/oculusvr/service.rs

```diff
@@ -14,7 +14,7 @@ const SERVICE_CLASS_NAME:&'static str = "com/rust/webvr/OVRService";
 
 pub struct OculusVRService {
     initialized: bool,
-    displays: Vec<OculusVRDisplayPtr>,
+    display: Option<OculusVRDisplayPtr>,
```

Comment:
Only one display?

Reply:
API seems to think so, and we use it that way